### DISCUSSION TITLE
trouble runnign the imgui-sdl example

### DISCUSSION THREAD

**Discussion by weather-tracker** [ocornut/imgui#5253] [New Users Build/Link/Run issues ONLY!]
I tried copying the imgui-sdl example cpp file.
most headers I have.
but i cannot find the imgui_imple_sdl.h or opengl3.h headers.
they are not included  in the vcpkg imgui package manager, or in the imgui download from git hub.
I have all others.

**Comment by weather-tracker** [comment]
found them in the github download under backends. 
how come they are not in the vcpkg download?

**Reply by PathogenDavid** [reply]
I am not especially familiar with vcpkg, but my understanding is you need to explicitly enable the `sdl2-binding` feature with `vcpkg install imgui[sdl2-binding]`

**Reply by weather-tracker** [reply]
when I run it now i get a exception thrown 
(void)SDL_atomicDecref(&s_mem.num_allocatoins);

I can tell it to continue then the boxup and another with a debug error
tell it retry to go to error location. error is IM_ASSERT(bd != NULL && "did you call ImGui_ImplOpenGL3_Init()?);

if I continue the program stops running.
it is the opengl3 example.

I do not know if I can run opengl3, the example said it should work with opengl2 unless I misunderstand it.

**Comment by weather-tracker** [comment]
works with opengl2 example
get first error expand to heap corruption 
continue and it runs.
